src/test/java/com/google/cloud/tools/appengine/operations/AuthTest.java

```diff
@@ -63,9 +62,7 @@ public void testLogin_withBadUser() {
       new Auth(gcloudRunner).login(testUsername);
       Assert.fail("Should have failed with bad user.");
     } catch (AppEngineException e) {
-      Assert.assertThat(
-          e.getMessage(), CoreMatchers.containsString("Invalid email address: " + testUsername));
-      // pass
+      Assert.assertTrue(e.getMessage().contains("Invalid email address: " + testUsername));
```

Comment:
could this not be converted to `org.hamcrest.MatcherAssert.assertThat` ?

Reply:
using http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/core/StringContains.html

```
import org.hamcrest.core.StringContains;

...
...

assertThat(e.getMessage(), StringContains.containsString("Invalid email address: " + testUsername));
```

Reply:
I can do that if you want, but the code's a little less clear and I suspect we'll have to  change it again in a few months anyway. The more we can do with the most fundamental classes the better.

Reply:
I would prefer assertThat, it provides a much better error message on failure. And I'm not convinced it's less readable.

Reply:
done